sig-contributor-experience/projects.md

```diff
@@ -0,0 +1,73 @@
+# Projects and Goals
+
+This is a list of the projects and goals currently underway with Contributor Experience. Please submit a PR if you are adding your project to this list. To introduce a new project, attend a weekly meeting or drop a note to us on the mailing list - details can be found on our [README](README.md). 
+
+Want to contribute? Take a look at this list and the "future" at the bottom. Thank you to all of our contributors for your hard work!
+
+## SIG Planning
+
+Project | Owner(s)/Lead(s) | Description | Q1, Q2, Later
+---|---|---|---
+[Charter](https://github.com/kubernetes/community/pull/1847) | SIG Leads | Create our first Charter iteration | Q1
+Goals and Projects | SIG Leads | Create projects.md | Q1
+de-SPOF Community Management | @castrojo | eg: YouTube management | Ongoing
+
+## Mentoring 
+Launch a multi-tier strategy test for group mentoring, mentors on demand (meet our contributors), diversity (outreachy), and google summer of code (students) that promote all levels of the [contributor ladder](community-membership.md).
```

Comment:
Instead of mentioning all items here and then in the table below..let's just say "Launch a multi-tier strategy test for following projects to promote all levels of the contributor ladder." (Also this will help in not worrying about keeping items here and in the table in sync.)